backend/core/src/map-layers/entities/map-layer.entity.ts

```diff
@@ -20,4 +20,9 @@ export class MapLayer {
   @Column()
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   jurisdictionId: string
+
+  @Column("jsonb")
+  @Expose()
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  featureCollection: any
```

Comment:
I feel like this creates some problems if we ever move to expose this as part of a DTO

as long as that won't be the case then this is okay!

Reply:
I wasn't happy about this as well. But my thought was that the `.entity` file is not needed in the Prisma backend and if we want to explicitly type it at that point it might be better to create a DTO in the new backend